examples/main/src/layer-examples.js

```diff
@@ -25,6 +25,12 @@ import {
 
 } from 'deck.gl';
 
+import {
```

Comment:
Could we split this file into three?
* official layer examples
* demo layer examples
* sample layer examples

The reason is that I'd like to keep the distinction between these layer types clear if possible.
I am also not sure we should include the demo layers here.  It kind of gives the impression that these are prepackaged layers ready to consume, rather than just examples provided for hacking. We should look at this live and decide.